### PR TITLE
Added auto directory creation for output folder.

### PR DESCRIPTION
**Pull request by sabihismail**
#1

### CODE REVIEW

File: lib/utils.rb [hunk 1]
```diff
@@ -44,7 +45,9 @@ def write_output_csv(output, directory, type)
                 return
             end
             output_file = "#{type.to_s}.csv"
-            CSV.open("#{OUTPUT_PATH}/#{directory}/#{output_file}", "w") do |csv|
+            dir_path = "#{OUTPUT_PATH}/#{directory}/"
```
Comment (Brainicism) [inline lib/utils.rb]: Would prefer to keep paths free of the `/` and concatenate them when necessary, like on line 50

File: lib/utils.rb [hunk 2]
```diff
@@ -44,7 +45,9 @@ def write_output_csv(output, directory, type)
...
+            FileUtils.mkdir_p dir_path
+            CSV.open(dir_path + "#{output_file}", "w") do |csv|
```
Comment (Brainicism) [inline lib/utils.rb]: Use string interpolation instead of concatenation here

File: lib/utils.rb [hunk 3]
```diff
@@ -44,7 +45,9 @@ def write_output_csv(output, directory, type)
                 return
             end
             output_file = "#{type.to_s}.csv"
-            CSV.open("#{OUTPUT_PATH}/#{directory}/#{output_file}", "w") do |csv|
+            dir_path = "#{OUTPUT_PATH}/#{directory}/"
+            FileUtils.mkdir_p dir_path
```
Comment (Brainicism) [inline lib/utils.rb]: Should check if the path already exists before creating it again

Comment (sabihismail) [reply lib/utils.rb]: It does :(

**Comment by Will-MingLun-Li** [comment]
Hey this PR is pretty epic

**Pull request merged**